package.json

```
@@ -57,31 +57,30 @@
   },
   "dependencies": {
     "@oclif/command": "^1.0",
-    "@oclif/config": "^1.12.11",
+    "@oclif/config": "^1.12",
     "@oclif/plugin-help": "^2.0",
-    "@stoplight/json": "^2.1.0",
-    "@stoplight/json-ref-resolver": "^1.5.0",
-    "@stoplight/types": "^7.0.1",
-    "@stoplight/yaml": "^2.3.0",
-    "ajv": "6.x.x",
-    "ajv-oai": "^1.1.1",
-    "chalk": "^2.4.2",
-    "jsonpath": "https://github.com/stoplightio/jsonpath#f1c0e9e634da2d45671b7639fa0a99afc807da17",
+    "@stoplight/json": "^2.2.2",
+    "@stoplight/json-ref-resolver": "^1.5",
+    "@stoplight/types": "^7.0",
+    "@stoplight/yaml": "^2.5",
+    "ajv": "^6",
+    "ajv-oai": "^1",
```

```suggestion
    "ajv-oai": "^1.1.1",
```
Suggest reverting this one as well.

---

Got it - do we need something from ajv-oai 1.1.1? just curious, can make the change as nothing downstream will likely use this dep, so can be more specific with it.

---

We need this commit https://github.com/BiteBit/ajv-oai/commit/6f7fa263b210a61db731d3b038f93ed0ccde369d.
It's on 1.1.0.